src/rgw/rgw_op.cc

```diff
@@ -829,6 +828,7 @@ int retry_raced_bucket_write(const DoutPrefixProvider *dpp, rgw::sal::Bucket* b,
 
 int RGWGetObj::verify_permission(optional_yield y)
 {
+  auto verify_permission_span = rgw_tracer.start_span("verify_permission", this->s->trace);
```

Comment:
`verify_permission()` and `execute()` are called on every op from `rgw_process_authenticated()`. i think these spans should start there, so we don't have to add them manually to each different kind of op like this

when an op needs access to this span (like when `RGWPutObj::execute()` adds the multipart id), we have a couple options..
1. add it as an argument to `void RGWOp::execute(const jspan& span)` and update every overload
2. have `req_state` store the current `span` instead of the initial `trace`, and have `rgw_process_authenticated()` overwrite that span before/after it calls `verify_permission()` and `execute()`

option 1) is a lot of work, so i tend to prefer 2). maybe that looks something like this?
```c++
  ldpp_dout(op, 2) << "verifying op permissions" << dendl;
  {
    auto span = rgw_tracer.start_trace("verify_permission", s->span);
    std::swap(span, s->span);
    ret = op->verify_permission(y);
    std::swap(span, s->span);
  } // verify_permission span destructs
```
that way every member function of `RGWOp` has access to the current span via `s->span`